css/style.css

```diff
@@ -1,19 +1,157 @@
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+body, footer {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
+
+/*Navigation*/
+.navbar{
+  background-color: white;
+  position: sticky;
+}
+.navbar .logo {
+  height: 60px;
+  margin-left: 30px;
+  float: left;
+}
+.navbar a{
+  float: left;
```

Comment:
Avoid using `float`. Use flexbox instead for layout.

Reply:
Updated - commits made